client/webpack.config.js

```diff
@@ -0,0 +1,99 @@
+var path = require('path');
+
+var webpack           = require('webpack');
+
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var CopyPlugin        = require('copy-webpack-plugin');
+var AssetsPlugin      = require('assets-webpack-plugin');
+var BrowserSyncPlugin = require('browser-sync-webpack-plugin');
+
+var production = process.env.NODE_ENV === 'production';
```

Comment:
Ok, nevermind, we can switch the env vars to `OPTIMIZE=true` and do it manually as we already done. Let's do it!